package.json

```diff
@@ -17,7 +17,7 @@
   "main": "lib/csdl2markdown.js",
   "dependencies": {
     "minimist": "^1.2.0",
-    "odata-csdl": "^0.2.4"
+    "odata-csdl": "github:oasis-tcs/odata-csdl-schemas"
```

Comment:
I changed this only temporarily so that I can push before we have updated npm. Will change back to `"odata-csdl": "^0.2.6"` before merging.

Reply:
0.2.6 is available on https://www.npmjs.com/package/odata-csdl